lib/engine/game/g_1893/step/buy_sell_par_shares.rb

```diff
@@ -58,6 +92,10 @@ def sell_adsk(bundle)
                                              price: price,
                                              allow_president_change: false)
           end
+
+          include ParAndBuy
+
+          include BuyMinor
```

Comment:
Could these two `include`s be moved to the top of the class definition?

Reply:
Ok